packages/lesswrong/components/ultraFeed/UltraFeedPostItem.tsx

```diff
@@ -229,28 +242,52 @@ const UltraFeedPostItem = ({
     settings.postTruncationBreakpoints
   ]);
 
-  if (
-    !post?._id 
-    || !post.contents
-    || !post.contents.htmlHighlight
-    || !post.contents.wordCount
-    || post.contents.wordCount <= 0
-  ) {
-     return <div>No post content found for post with id: {post._id}</div>; 
+  const handleCollapse = () => {
+    setResetSig((s) => s + 1);
+  };
+
+  const handleOpenDialog = useCallback((params?: {textFragment?: string}) => {
```

Comment:
Are we no longer passing in any `params` in any of the callsites?

Reply:
UltraFeedPostItem and UltraFeedCommentItem pass the callback to FeedContentBody, which does pass the param

So we are